### ISSUE TITLE
rollback does not work

### ISSUE THREAD

**Issue by Hazugg** [Engineer1986/TheEndProtector#7]
rollback doesn't work and this message appears in console when it is executed

![image](https://user-images.githubusercontent.com/31551514/113482043-6bba2800-9462-11eb-94a6-b4265bc9d933.png)

**Comment by Engineer9736** [comment]
Hi, are you running the latest CoreProtect? Version 19.5 ? And did you fully stop and start (so no reload) the server after adding the jar to the plugins directory?

Do you maybe run any modifications to the end causing "world_the_end" having a different name?

**Comment by Hazugg** [comment]
Yes, I am running the latest version of coreprotec precisely 19.5 and my world "world_the_end" actually has another name, it has the original name of my main world.

**Comment by Hazugg** [comment]
and if i stopped and reloaded the server and that still persists.

**Comment by Hazugg** [comment]
any way to be able to fix that? I really need this on my server.

**Comment by Engineer9736** [comment]
Hi, i just realize, in the initial the-end-world the dragon is alive right away, while the rollback timestamp is set when the dragon is spawned one time. This could explain the null pointer exception. If this is the case, the problem should be gone as soon as a dragon has been respawned. So try /killdragon and /spawndragon, then it should work. If not then i'll look into it later, got to finish other things first unfortunately 🙂

**Comment by Hazugg** [comment]
still not working, I'll wait when I can fix it, thanks!
